metadrive/manager/scenario_traffic_manager.py

```diff
@@ -162,6 +163,10 @@ def sdc_object_id(self):
     def current_scenario_length(self):
         return self.engine.data_manager.current_scenario_length
 
+    @property
+    def vehicles(self):
```

Comment:
As there are more objects in a traffic scenario now, drawing only vehicles is not enough. It would be better to draw all objects including cones, barriers, pedestrians, and bikes. So I suggest drawing all objects like what we did in the Top-Down renderer. This can be done by accessing all objects via `engine._spawned_objects` or `engine.get_objects()` and draw them.

Reply:
I have done so now!